src/components/GameClock.cpp

```diff
@@ -1,24 +1,28 @@
 #include "GameClock.hpp"
+#include <Math/BsMath.h>
 #include <RTTI/RTTI_GameClock.hpp>
 #include <Scene/BsSceneObject.h>
-#include <Math/BsMath.h>
 
 namespace REGoth
 {
   constexpr float SECONDS_IN_A_MINUTE = 60;
   constexpr float SECONDS_IN_AN_HOUR  = 60 * SECONDS_IN_A_MINUTE;
   constexpr float SECONDS_IN_A_DAY    = 24 * SECONDS_IN_AN_HOUR;
-  constexpr float CLOCK_SPEED_FACTOR  = SECONDS_IN_A_DAY/6000; // see https://forum.worldofplayers.de/forum/threads/396326-Tipp-Tageweise-springen-%28auch-zur%C3%BCck%29?p=6231841&viewfull=1#post6231841
+  // see
+  // https://forum.worldofplayers.de/forum/threads/396326-Tipp-Tageweise-springen-%28auch-zur%C3%BCck%29?p=6231841&viewfull=1#post6231841
+  // constexpr float CLOCK_SPEED_FACTOR = SECONDS_IN_A_DAY / 6000;
+  constexpr float CLOCK_SPEED_FACTOR = SECONDS_IN_A_DAY / 60;
```

Comment:
I think you didn't intend to commit this change?

Reply:
Ah, darn it, I didn't.